RELEASING.adoc

```diff
@@ -1,15 +1,27 @@
 = Releasing Kiali
 
-Kiali is released using a Jenkins Pipeline. If you need to release
-a new version of Kiali, please ask to a member of the QE team to
-trigger the release for you. Alternatively, ask the QE team for
-access to the Jenkins instance and trigger the release by yourself.
-
-Currently, the Pipeline supports only _minor_ and _snapshot (end-of-week)_
-releases and will release the back-end and the front-end together. If
-you need to do a patch or a major release, you'll need to
+Kiali is released using a Jenkins Pipeline, which is scheduled to
+run each Friday between 2pm and 3pm of America/Merida timezone.
+Every three Fridays a minor release is built. The other Fridays,
+a snapshot release is built.
+
+If you need to make an extraordinary release, please ask
+to a member of the QE team to trigger the release for you.
+Alternatively, ask the QE team for access to the Jenkins instance
+and trigger the release by yourself.
+
+If you don't manage to run the Jenkins job, you could
```

Comment:
`If you do not have access to Jenkins, or want to test the release locally, you can run it on your machine.`

Maybe?